go-controller/pkg/ovn/egressfirewall.go

```diff
@@ -27,10 +28,14 @@ import (
 const (
 	egressFirewallAppliedCorrectly = "EgressFirewall Rules applied"
 	egressFirewallAddError         = "EgressFirewall Rules not correctly added"
-	// egressFirewallACLExtIdKey external ID key for egress firewall ACLs
-	egressFirewallACLExtIdKey    = "egressFirewall"
-	egressFirewallACLPriorityKey = "priority"
-	aclDeleteBatchSize           = 1000
+	aclDeleteBatchSize             = 1000
+	aclUpdateBatchSize             = 10000
```

Comment:
why 10k?

Reply:
half-random, I can try some tests to be more specific